stdlib/public/runtime/ErrorObject.mm

```diff
@@ -334,6 +332,10 @@ static id _swift_bridgeErrorToNSError_(SwiftError *errorObject) {
   NSDictionary *userInfo =
     swift_stdlib_getErrorUserInfoNSDictionary(value, type, witness);
 
+  // Never produce an empty userInfo dictionary.
+  if (!userInfo)
+    userInfo = SWIFT_LAZY_CONSTANT(@{});
```

Comment:
I eliminated the other place I know about, but I do want to revisit this because on modern platforms we don't need the dispatch_once at all for empty collection literals.